src/main/resources/application.yml

```diff
@@ -1,12 +1,33 @@
+version: v0.2.19
+
+---
+spring:
+  profiles: production
+  data:
+    rest:
+      return-body-on-create: true
+      return-body-on-update: true
+  jpa:
+    show-sql: false
 server:
-  port: ${port:8081}
+  port: ${port:80}
```

Comment:
This config is for `production` mode and the one below is for `development` mode. Application can only run in one or the other.

* In `production` if not supplied it will default to `80`.
* In `development` if not supplied it will default to `8081`